docs/product-editor-development/README.md

```diff
@@ -0,0 +1,22 @@
+# Developing extensions for the product editor
+
+> ⚠️ **Notice:** This documentation is currently a **work in progress**. While it's open to the public for transparency and collaboration, please be aware that some sections might be incomplete or subject to change. We appreciate your patience and welcome any contributions!
```

Comment:
Suggestion: Remove the `While it's open to the public for transparency and collaboration.` I don't think it really adds much to the text, and makes it more wordy.